phpunit/class-wp-theme-json-test.php

```diff
@@ -1598,4 +1615,94 @@ public function test_get_stylesheet_handles_custom_css() {
 		$custom_css = 'body { color:purple; }';
 		$this->assertEquals( $custom_css, $theme_json->get_stylesheet( array( 'custom-css' ) ) );
 	}
+
+	public function test_allows_custom_css_for_users_with_caps() {
+		wp_set_current_user( self::$administrator_id );
+
+		// Explicitly grant 'edit_css' capabilities.
+		$grant_edit_css_cap = function( $caps, $cap ) {
+			if ( 'edit_css' === $cap ) {
+				$caps = array( 'edit_theme_options' );
+			}
+			return $caps;
+		};
+		add_filter( 'map_meta_cap', $grant_edit_css_cap, 10, 2 );
```

Comment:
Instead of this, in the `set_up_before_class()` fixture, you can add this code which grants this specific admin Super Admin access.

```php
if ( is_multisite() ) {
	grant_super_admin( self::$administrator_id );
}
```

Let me push an update to see if it resolves the issue.

Reply:
Done in https://github.com/WordPress/gutenberg/pull/47062/commits/35eb0418a8eec5ca3fac8c41911a4f69ff82b959 and https://github.com/WordPress/gutenberg/pull/47062/commits/f05482f38c223f03ed9663953acf7ee38f99f9d9. PHPUnit tests are passing ✅

Reply:
Thank you, @hellofromtonya 🙇